src/ionization.jl

```diff
@@ -12,9 +12,24 @@ function ionization_collisions!(f_out, fvec_in, moments, n_ion_species,
         n_neutral_species, vpa, z, r, composition, collisions, nz, dt)
 
     if moments.evolve_density
-        error("Ionization collisions not currently supported for anything other than the standard drift kinetic equation: Aborting.")
+		@loop_s is begin
```

Comment:
replace tabs with spaces
```suggestion
        @loop_s is begin
```